### PR TITLE
feat(curriculum): add CSS Backgrounds and Borders Quiz

### LINKED ISSUE #56180: [HACKTOBERFEST 2024] - CSS Backgrounds and Borders Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the CSS Backgrounds and Borders quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/css-backgrounds-and-borders`.

**All quiz questions need to relate to one of these topics:**

- list-style properties
- link styles and states
- background images (size, repeat, etc)
- accessibility concerns for backgrounds
- borders around images

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by rakshixh**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56180

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md [hunk 1]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which CSS property is used to specify the image to be used as a list item marker?
 
 #### --distractors--
 
-Placeholder distractor 1
+list-style-position
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md:24]:
```suggestion
`list-style-position`
```

you need to use backticks around anything code, even when they are the only word in an answer/distractor

**Review by majestic-owl448 — COMMENTED** [review]
I gave a cursory review, you need to use backticks more. I left a comment that you can use to improve many other questions

File: curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md (hunk 1 above)
Comment (rakshixh) [reply curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md:24]: Oh my bad, I have added that for code which are in questions but didn't do the same for options. Anyhow will commit the suggested changes, please give some time. Thank you!

Comment (rakshixh) [reply curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md:24]: Which of the following is NOT a valid value for the `list-style-type` property?

#### --distractors--

decimal

---

lower-greek

---

square

#### --answer--

lower-arabic


@ilenia-magoni I just wanted to know if i should use ` ` for values as well. As you can see the above example in that options are values so should I put them as well inside the ` ` ?

**Comment by rakshixh** [comment]
@ilenia-magoni Hey I have added backticks for all code snippets including for css values. Please review it and let me know if any other changes u would like me to do. 

Thank you!

File: curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md (hunk 1 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md]: exactly those needs to go in backticks

**Review by naomi-lgbt — APPROVED** [review]

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @rakshixh. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by rakshixh** [comment]
Hey @naomi-lgbt ,
Please find my discord username @rakshixh to assign me a role on discord channel.
Thank you!

**Pull request merged**